src/components/Layout/Editor/Editor.tsx

```diff
@@ -33,9 +33,14 @@ const Editor = ({ onChange, markdown }: EditorType) => {
       ?.getInstance()
       .getMarkdown()
       .toString() as string
+
     if (markdown !== currentMd) onChange(currentMd)
   }, [editorRef.current?.getInstance().getMarkdown(), markdown, onChange])
 
+  useEffect(() => {
+    if (markdown) editorRef.current?.getInstance().setMarkdown(markdown)
+  }, [])
```

Comment:
```suggestion
  }, [markdown])
```

probably we want to execute the effect on markdown changes?